app/eq.py

```diff
@@ -195,22 +195,21 @@ def camel_to_snake(s):
     @staticmethod
     def build_display_address(sample_attributes):
         """
-        Build `display_address` value by appending not-None (in order) prem values of sample attributes
+        Build `display_address` value by appending not-None (in order) values of sample attributes
 
-        :param sample_attributes: dictionary of attributes with Prem1, Prem2, Prem3, Prem4 keys present
+        :param sample_attributes: dictionary of address attributes
         :return: string of a single prem value or a combination of two
         """
         display_address = ''
-        for prem_key in ['Prem1', 'Prem2', 'Prem3', 'Prem4']:  # retain order of Prem values
-            try:
-                val = sample_attributes[prem_key]
-            except KeyError:
-                raise InvalidEqPayLoad(f"{prem_key} missing from sample attributes")
+        for prem_key in ['Prem1', 'Prem2', 'Prem3', 'Prem4', 'District', 'PostTown', 'Postcode']:  # retain order of Prem values
```

Comment:
Suggested change to comment:

`# retain order of address attributes`

Reply:
Done